sdks/java/io/tika/src/main/java/org/apache/beam/sdk/io/tika/TikaIO.java

```diff
@@ -60,85 +92,96 @@
 public class TikaIO {
 
   /**
-   * A {@link PTransform} that parses one or more files and returns a bounded {@link PCollection}
-   * containing one element for each sequence of characters reported by Apache Tika SAX Parser.
+   * A {@link PTransform} that matches and parses the files
+   * and returns a bounded {@link PCollection} of {@link ParseResult}.
    */
-   public static Read read() {
-     return new AutoValue_TikaIO_Read.Builder()
-        .setQueuePollTime(Read.DEFAULT_QUEUE_POLL_TIME)
-        .setQueueMaxPollTime(Read.DEFAULT_QUEUE_MAX_POLL_TIME)
+  public static ParseAll parseAll() {
+    return new AutoValue_TikaIO_ParseAll.Builder()
         .build();
-   }
+  }
 
-   /** Implementation of {@link #read}. */
-  @AutoValue
-  public abstract static class Read extends PTransform<PBegin, PCollection<String>> {
-    private static final long serialVersionUID = 2198301984784351829L;
-    public static final long DEFAULT_QUEUE_POLL_TIME = 50L;
-    public static final long DEFAULT_QUEUE_MAX_POLL_TIME = 3000L;
+  /**
+   * A {@link PTransform} that accepts a bounded {@link PCollection} of {@link ReadableFile}
+   * and returns a bounded {@link PCollection} of {@link ParseResult}.
```

Comment:
This is not necessarily true: the collection can be unbounded, depending on whether the input collection is bounded. Just say that it returns a collection of ParseResult.